### DISCUSSION TITLE
problem with keys in blackarch repo

### DISCUSSION THREAD

**Discussion by EsamLasheen** [BlackArch/blackarch#3910] [Support]
![image](https://github.com/BlackArch/blackarch/assets/55217611/c7470a6b-e2fa-4b53-bc7d-028734dd7502)
 this bug in theharvester python-uvicorn python-plotly

**Comment by noraj** [comment]
Have you tried `pacman -Syu` before installing? Have you read _[Why do I get invalid keyring signature?](https://blackarch.org/faq.html#collapseFour)_ in the FAQ?

**Comment by EsamLasheen** [comment]
Yes, by the way, I'm from Egypt. Maybe this is a problem?

**Comment by noraj** [comment]
The quickest way to get support is to ask the community on Matrix: https://blackarch.org/community.html

**Comment by ikstream** [comment]
> Yes, by the way, I'm from Egypt. Maybe this is a problem?

This shouldn't cause any issues. As you haven't reacted on any responses in the matrix channel, to your request there, I assume your issue is fixed?
